filebeat/module/traefik/access/ingest/pipeline.json

```diff
@@ -3,21 +3,23 @@
     "processors": [
         {
             "grok": {
-                "field": "traefik.access.message",
+                "field": "message",
                 "patterns": [
-                    "(?:%{NUMBER:traefik.access.body_sent.bytes}|-)( \"%{DATA:traefik.access.referrer}\")?( \"%{DATA:traefik.access.agent}\")?(?:%{NUMBER:traefik.access.request_count}|-)?( \"%{DATA:traefik.access.frontend_name}\")?( \"%{DATA:traefik.access.backend_url}\")?"
+                    "%{IPORHOST:traefik.access.remote_ip} %{GREEDYDATA:traefik.access.user_identifier} %{GREEDYDATA:traefik.access.user_name} \\[%{HTTPDATE:traefik.access.time}\\] \"%{WORD:traefik.access.method} %{DATA:traefik.access.url} HTTP/%{NUMBER:traefik.access.http_version}\" %{NUMBER:traefik.access.response_code:int} %{NUMBER:traefik.access.body_sent.bytes:int}( \"%{GREEDYDATA:traefik.access.referrer}\" \"%{DATA:traefik.access.agent}\" %{NUMBER:traefik.access.request_count:int} \"%{DATA:traefik.access.frontend_name}\" \"%{DATA:traefik.access.backend_url}\" %{NUMBER:traefik.access.duration:int}ms)?"
```

Comment:
It seems now dissect and grok do the same at least the initial part. Is that intentional?

Reply:
Yes. At least it was the only way to parse all possible content without deleting dissect part.

Reply:
Tried quite some time without duplicating the initial part but it wasn't working

Reply:
I tried the following and it seems to work:

```
"field": "traefik.access.message",
                "patterns": [
                    "%{NUMBER:traefik.access.body_sent.bytes:int}( \"%{GREEDYDATA:traefik.access.referrer}\" \"%{DATA:traefik.access.agent}\" %{NUMBER:traefik.access.request_count:int} \"%{DATA:traefik.access.frontend_name}\" \"%{DATA:traefik.access.backend_url}\" %{NUMBER:traefik.access.duration:int}ms)?"
                ],
```
do you try it based on message or treafik.access.message?

Reply:
`traefik.access.message` was giving me errors of not being able to process from the request bytes count field and I only managed to make it work with this approach

Reply:
I tested the above and it seems to work on my machine?